cli/tests/node_compat/test.ts

```diff
@@ -78,7 +79,7 @@ for await (const path of testPaths) {
       });
       const { code, stdout, stderr } = await command.output();
 
-      if (stdout.length) console.log(decoder.decode(stdout));
+      if (stdout.length && hasFilters) console.log(decoder.decode(stdout));
```

Comment:
Can you make it display the output if code is non-zero?

Reply:
Good point. Updated!